alws/schemas/test_repository_schema.py

```diff
@@ -45,6 +45,8 @@ class TestRepositoryCreate(BaseModel):
     url: AnyHttpUrlString
     tests_dir: str
     tests_prefix: typing.Optional[str] = None
+    team_id: int
+    owner_id: int
```

Comment:
Why do we need `owner_id` when we create a test repository? I don't see where we use this variable

Reply:
You are correct, I removed it and it worked fine

Reply:
@maccelf It's true that we shouldn't really need it, but look at `can_perform` implementation, which  [does the following](https://github.com/AlmaLinux/albs-web-server/blob/e842886770a6302053092d60e5b039b249d41835/alws/perms/authorization.py#L38C5-L38C31). If the object, in this case, the `test_repository` doesn't have an `owner_id`, `can_perform` will fail while trying to access `obj.owner.id`.  Or maybe I'm missing something here.

However, I'd say that we can stop sending it from frontend (is related to [this review comment](https://github.com/AlmaLinux/albs-frontend/pull/475#discussion_r1629717744) of the frontend side of this PR) and manually update the create payload with the current user that is actually performing the action.

Reply:
Ok, looking at this again, this is what we actually do when creating new products:
* frontend sends user_id -> https://github.com/AlmaLinux/albs-frontend/blob/master/src/pages/CreateProduct.vue#L127
* productCreate schema has owner_id -> https://github.com/AlmaLinux/albs-web-server/blob/master/alws/schemas/product_schema.py#L15
Said that, I'm not sure this is right, maybe we should use the team's owner for new created resources (either products or test_repositories). @maccelf, any thoughts on that?

Reply:
> we should use the team's owner for new created resources (either products or test_repositories).

As I remember it's a core idea of RBAC model in ALBS when we was making a prototype.

Reply:
IDK, Is it right if the non-owner of the team creates a test repository and then he will not be an owner of the test repository? Thats weird

Reply:
Well, that's quite a fair point. In my head, I'd say the resource should belong to the team, as the user is creating it on behalf of the team, but `owner_id` can't be a team, hence my proposal. But yes, let's follow the product's owner_id pattern (the user owns the resource). In that case, we do need this in the payload and this comment can be closed, right?

Reply:
Yes, I think we can keep it as it is